lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -237,6 +207,10 @@ states:
         targetState: NINO_START_PAGE
       end:
         targetState: F2F_PYI_POST_OFFICE
+        checkIfDisabled:
```

Comment:
Were we saying we didn't need any routing for f2f? If so then we should probably just send them to:
```
targetJourney: INELIGIBLE
targetState: INELIGIBLE_SKIP_MESSAGE
```

Reply:
Hm possible, @blakeyp  Can you pls confirm if we need this check ? and need to send user to diff target State for f2f ?

Reply:
For now we do still need to handle F2F being disabled same as the P2 journey